src/from_markdown.js

```diff
@@ -81,8 +81,8 @@ class MarkdownParseState {
   }
 }
 
-function attrs(given, token) {
-  return given instanceof Function ? given(token) : given
```

Comment:
This breaks backwards compatibility. Are you trying to align it with the prop names in `ParseRule`? Could you please also keep the `instanceof Function` test, so that we don't break people's code.